pyproject.toml

```diff
@@ -1,4 +1,47 @@
-[build-system]
-requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=3.4", "Cython"]
+[tool.poetry]
+name = "pegasus"
+version = "1.4.4"
```

Comment:
Is it possible to dynamically get GitHub tag-based version, instead of manually setting the version number? Just like setting the `use_scm_versions` option to `True` in `setup` function of `setup.py`.

Reply:
thats a good point, ill check